calc/package-lock.json

```diff
@@ -1,18 +1,18 @@
 {
   "name": "@smogon/calc",
-  "version": "0.7.0",
+  "version": "0.8.0",
```

Comment:
I don't know why the version in the package-lock.json changes, I just edited the `package.json`, then did `rm -rf node_modules` and `npm i` and the version modified automatically.

Reply:
Maybe someone forgot to commit a package-lock change? This happened to me too